msm/src/circuit_design/witness.rs

```diff
@@ -352,6 +362,15 @@ impl<
             witness.cols[N_REL + N_DSEL + i] = self.fixed_selectors[i].clone();
```

Comment:
Maybe it is modified later in https://github.com/o1-labs/proof-systems/pull/2249/files, but we should be sure that `set_fixed_selectors` has been called before.

Reply:
Agreed. I want to rework the design later on to remove `set_fixed_selectors` altogether and take selectors from some trait for the column.